test/cppsuite/configs/search_near_02_default.txt

```diff
@@ -0,0 +1,19 @@
+# Configuration for search_near_02.
```

Comment:
Just nudging that the config needs updating :)

Reply:
Updated in ccc315274e11d8811daaadce5e2753e3b7f6945d